pkg/kafka/kafka_util.go

```diff
@@ -82,3 +83,18 @@ func ValidateSearchInput(val interface{}) error {
 
 	return kafkaerr.InvalidSearchValueError(search)
 }
+
+// ValidateNameIsAvailable checks if a kafka instance with the given name already exists
+func ValidateNameIsAvailable(api kafkamgmtv1.DefaultApi) func(v interface{}) error {
+	return func(v interface{}) error {
+		name, _ := v.(string)
+
+		_, httpRes, _ := GetKafkaByName(context.Background(), api, name)
```

Comment:
Getting a lint error here with the message: `response body must be closed (bodyclose)`. Calling close() method does remove this message but causes other issues. Any suitable workaround for this is appreciated.